tools/spelling/spelling_dictionary.txt

```diff
@@ -52,6 +52,7 @@ DSR
 DSS
 EBADF
 ECH
+enable_lazyinit_stats
```

Comment:
I feel like we don't generally have all the config settings in the dictionary file. I suspect you are using this in a comment in a way that's not usually done, forcing this spelling hack.